src/com/oltpbenchmark/schema/TPCCTableSchemas.java

```diff
@@ -0,0 +1,159 @@
+                .column("c_ytd_payment", "float NOT NULL")
+                .column("c_payment_cnt", "int NOT NULL")
+                .column("c_delivery_cnt", "int NOT NULL")
+                .column("c_street_1", "varchar(20) NOT NULL")
+                .column("c_street_2", "varchar(20) NOT NULL")
+                .column("c_city", "varchar(20) NOT NULL")
+                .column("c_state", "char(2) NOT NULL")
+                .column("c_zip", "char(9) NOT NULL")
+                .column("c_phone", "char(16) NOT NULL")
+                .column("c_since", "timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP")
+                .column("c_middle", "char(2) NOT NULL")
+                .column("c_data", "varchar(500) NOT NULL")
+                .primaryKey("((c_w_id,c_d_id) HASH,c_id)")
+                .partitionKey("(c_w_id)")
+                .build(),
+            TableSchema.Builder()
+                .name(TPCCConstants.TABLENAME_DISTRICT)
+                .column("d_w_id", "int NOT NULL")
+                .column("d_id", "int NOT NULL")
+                .column("d_ytd", "decimal(12,2) NOT NULL")
+                .column("d_tax", "decimal(4,4) NOT NULL")
+                .column("d_next_o_id", "int NOT NULL")
+                .column("d_name", "varchar(10) NOT NULL")
+                .column("d_street_1", "varchar(20) NOT NULL")
+                .column("d_street_2", "varchar(20) NOT NULL")
+                .column("d_city", "varchar(20) NOT NULL")
+                .column("d_state", "char(2) NOT NULL")
+                .column("d_zip", "char(9) NOT NULL")
+                .primaryKey("((d_w_id,d_id) HASH)")
+                .partitionKey("(d_w_id)")
+                .build(),
+            TableSchema.Builder()
+                .name(TPCCConstants.TABLENAME_ITEM)
+                .column("i_id", "int NOT NULL")
+                .column("i_name", "varchar(24) NOT NULL")
+                .column("i_price", "decimal(5,2) NOT NULL")
+                .column("i_data", "varchar(50) NOT NULL")
+                .column("i_im_id", "int NOT NULL")
+                .primaryKey("(i_id)")
+                .build(),
+            TableSchema.Builder()
+                .name(TPCCConstants.TABLENAME_WAREHOUSE)
+                .column("w_id", "int NOT NULL")
+                .column("w_ytd", "decimal(12,2) NOT NULL")
+                .column("w_tax", "decimal(4,4) NOT NULL")
+                .column("w_name", "varchar(10) NOT NULL")
+                .column("w_street_1", "varchar(20) NOT NULL")
+                .column("w_street_2", "varchar(20) NOT NULL")
+                .column("w_city", "varchar(20) NOT NULL")
+                .column("w_state", "char(2) NOT NULL")
+                .column("w_zip", "char(9) NOT NULL")
+                .primaryKey("(w_id)")
+                .partitionKey("(w_id)")
+                .build()
+            ).collect(Collectors.toMap(TableSchema::name, e -> e));
+    
+    public static String getInsertDml(String tablename) {
```

Comment:
Current API looks strange.

class `Table` has `getDropDdl` and `getCreateDdl` but doesn't have `getInsertDml`. Is it possible to move `getInsertDml` into `Table` class to avoid blurring of responsibility.